domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/opprett/OpprettNySøknadsbehandling.kt

```diff
@@ -27,9 +27,13 @@ fun Sak.opprettNySøknadsbehandling(
     saksbehandler: NavIdentBruker.Saksbehandler,
 ): Either<Sak.KunneIkkeOppretteSøknadsbehandling, Tuple4<Sak, NySøknadsbehandling, Søknadsbehandling.Vilkårsvurdert.Uavklart, StatistikkEvent.Behandling.Søknad.Opprettet>> {
     if (!harIngenÅpneBehandlinger()) {
-        // TODO jah: Endre denne når de nye begrensningene er på plass for opprettelse og iverksettelse.
+        // TODO jah: Endre denne når de nye begrensningene er på plass for opprettelse og iverksettelse av søknadsbehandling/revurdering/regulering.
         return Sak.KunneIkkeOppretteSøknadsbehandling.HarÅpenBehandling.left()
     }
+    if (harÅpenSøknadsbehandling()) {
```

Comment:
Denne treffes vel av `!harIngenÅpneBehandlinger `?

Reply:
// Denne dekkes inntil videre av `harÅpenBehandling()` over. Conditionen over skal slettes når vi har lagt inn alle begrensninger på opprett/oppdater/iverksett for regulering/revurdering/søknadsbehandling.